jenkins/automation/kubernetes/nightly-build-and-deploy.groovy

```diff
@@ -142,10 +142,11 @@ pipeline {
                 junit allowEmptyResults: true, testResults: '*report.xml'
                 echo "${env.cortxCluster_status}"
                 echo "${env.qaSanity_status}"
-                if ( "${env.cortxCluster_status}" == "SUCCESS" && "${env.qaSanity_status}" == "SUCCESS" ) {
+                if ( "${env.cortxCluster_status}" == "SUCCESS" && "${env.qaSanity_status}" == "SUCCESS" || params.EMAIL_RECIPIENTS == "ALL") {
```

Comment:
Instead of changing anything here can we keep `mailRecipients` deciding logic at https://github.com/Seagate/cortx-re/blob/main/jenkins/automation/kubernetes/nightly-build-and-deploy.groovy#L190 block only
Additional conditions will be the entire build status.

Reply:
Addressed in latest check-in